lox_services/persistence/storage/storage.py

```diff
@@ -123,7 +124,8 @@ def download_file_from_url(
 
     bucket = path.split("/")[0]
     blob = path.split("/", 1)[1]
-    file_name = blob.rsplit("/", 1)[1]
+    file_name = remove_timestamp_from_file_name(blob.rsplit("/", 1)[1])
```

Comment:
will it also work for the one without timestamp ?

Reply:
@mathisbrrt Yes